components/PostCard.tsx

```diff
@@ -1,13 +1,43 @@
+import {
+  isPostADraft,
+  isPostInTheFuture,
+} from "helpers/checkOfDraftOrFuturePost";
 import { PostDocumentWithoutContent } from "interfaces";
 import Image from "next/image";
 import Link from "next/link";
 import { useAppDispatch } from "redux/typesHooks";
 
 import { setTags } from "../redux/slices/selectedTags";
+
 interface Props {
   post: PostDocumentWithoutContent;
 }
 
+export const DraftPostMark = () => (
+  <div className="drt-post">
+    <div className="trn-drt trn trn-i">
+      <span className="trn-txt-drt trn-txt">draft</span>
+    </div>
+    <div className="sm-trns-drt">
+      <div className="sm-trns-i-drt-first trn sm-trns-i-drt sm-trns-i"></div>
+      <div className="sm-trns-i-drt-sec trn sm-trns-i-drt sm-trns-i"></div>
+    </div>
+  </div>
+);
+
+export const FuturePostMark = () => (
+  <div className="fut-post">
+    <div className="trn-fut trn trn-i">
+      <span className="trn-txt-fut trn-txt">future</span>
+      <span className="trn-txt-fut-post trn-txt"> post</span>
+    </div>
+    <div className="sm-trns-fut">
+      <div className="sm-trns-i-fut-first trn sm-trns-i-fut sm-trns-i"></div>
+      <div className="sm-trns-i-fut-sec trn sm-trns-i-fut sm-trns-i"></div>
+    </div>
+  </div>
+);
```

Comment:
Same story, just git revert commit with renaming

Reply:
Wrote above about it

Reply:
@1alexvash

Reply:
answered